manual.rb

```diff
@@ -0,0 +1,80 @@
+# load it to a pry instance and run `simple_test`
```

Comment:
This will be removed in the final version?

Reply:
Yep